docs/security/email-link-protection.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Protect email link sign-ins and sign-ups
+description: Learn how to protect email link sign-ins and sign-ups.
+---
+
+# Protect email link sign-ins and sign-ups
+
+Clerk provides an additional setting to improve security for email link sign-ins and sign-ups. When this setting is enabled, only email links opened on the same device and browser that a sign-in or sign-up was initiated on will successfully authenticate the user.
+
+Enabling this protection means that the following scenario cannot happen:
+
+1. A malicious actor knows the email of a user and requests an email link to sign-in or sign-up.
+2. The user opens the email and visits the link, or the user's email provider follows links in emails for phishing or spam protection.
+3. The malicious actor is able to gain access to the user's account because the email link has been opened successfully.
+
+Disabling this protection means that your users are vulnerable to the scenario above, but they will be able to sign-in or sign-up with email link across devices and browsers. For example, initiating an email link sign in on a laptop and then opening the email link on a phone to sign in successfully.
+
+With same device and browser protection on, users will see the following warning if an email link for sign-in or sign-up was opened on a different device or browser:
+
+<Images
+width={1512}
+height={914}
+src="/docs/images/security/email-link_require-same-client.webp"
+alt="Error message displayed if email link is opened in a different device or browser if email link same device/browser protection is enabled"
+/>
+
+To configure this security setting, go to [Email, Phone, and Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) section of the Clerk Dashboard.
+
+To enable this protection for sign-ups, go to **Contact information** > **Email address**. Open the modal and make sure **Require the same device and browser** is enabled under the **Email verification link** checkbox.
+
+To enable this protection for sign-ins, go to **Authentication strategies** > **Email verification link**. Ensure **Require the same device and browser** is toggled on in the configuration modal.
```

Comment:
Is there a way I can test this in the Dashboard? It's not currently there, and I don't see it in staging either.

```suggestion
To configure this security setting, navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) and in the navigation sidebar, select **Email, Phone, Username**.

To enable this protection for **sign-ups**:
1. In the **Contact information** section, next to **Email address**, select the settings icon.
1. Under the **Email verification link** checkbox, ensure **Require the same device and browser** is enabled.

To enable this protection for **sign-ins**:
1. In the **Authentication strategies** section, next to **Email verification link**, select the settings icon.
1. Ensure **Require the same device and browser** is toggled on.
```

Reply:
It's enabled in staging now! https://clerkinc.slack.com/archives/CHZ1FBBEG/p1717534767333909

I also updated this section a bit. I realized the previous wording was implying the setting could be turned on for sign-ins _only_, for example, but it's the same setting for both sign-ins and sign-ups. You can't turn it on only for one or the other. Lmk what you think!